examples/gitlab-asg/Makefile

```diff
@@ -40,8 +35,12 @@ install-cfssl:
 
 ## Use AWS cli tools to upload the SSL cert to AWS
 upload-tls-certs:
-	@AWS_DEFAULT_REGION=us-east-1 aws acm import-certificate --certificate file://gitlab.pem --private-key file://gitlab-key.pem --certificate-chain file://ca.pem
-
+	@. ./vars.env && envsubst < upload-gen-cert.tpl > upload-gen-cert.sh
+	@. ./vars.env && envsubst < delete-gen-cert.tpl > delete-gen-cert.sh
+	@chmod +x ./upload-gen-cert.sh ./delete-gen-cert.sh
+	@./upload-gen-cert.sh
+	@echo "ssl_arn = " `cat  upload-gen-cert-arn.txt` >> terraform.tfvars
```

Comment:
Appending to `terraform.tfvars` here assumes that `make render-tfvars` has already been run, and also means that re-running `make render-tfvars` later will "erase" the value written by this step (so it has to be re-run).  Perhaps better would be to make this file an _input_ to `make render-tfvars`, and move the step of running that command to be after `make upload-tls-certs`.

Reply:
Fixed along with next comment.